src/components/routes/Listing/Listing/ListingInformation.tsx

```diff
@@ -9,11 +9,11 @@ import GoogleMaps from 'shared/GoogleMaps';
 import LazyImage from 'shared/LazyImage';
 import ListItem from 'shared/ListItem';
 import Svg from 'shared/Svg';
-import { dateToYear } from 'utils/formatDate';
 import { formatAddress  } from 'utils/formatter'
 import Portal from 'components/shared/Portal';
 import ContactHostForm from 'components/shared/ContactHostForm';
 import { ToggleProviderRef, ToggleProvider } from 'components/shared/ToggleProvider';
+import { format } from 'date-fns';
```

Comment:
i think you can cherry pick by doing `import format from 'import format from `date-fns/format';`

Reply:
🙏